src/distrib/dickeyfuller.rs

```diff
@@ -14,27 +14,75 @@
 
 use num_traits::Float;
 
-use super::AlphaLevel;
+use super::{AlphaLevel, CalculationError, Regression};
 
 /// Approximate Dickey-Fuller distribution for specific alpha levels
 /// for constant, no trend: $Δy_i = β_0 + β_1*y_{i-1} + ε_i$
 /// https://www.real-statistics.com/statistics-tables/augmented-dickey-fuller-table/
-pub fn constant_no_trend_critical_value<F: Float>(sz: usize, alpha: AlphaLevel) -> F {
+pub fn constant_no_trend_critical_value<F: Float>(
+    sz: usize,
+    alpha: AlphaLevel,
+) -> Result<F, CalculationError> {
     let (t, u, v, w) = match alpha {
         AlphaLevel::OnePercent => (-3.43035, -6.5393, -16.786, -79.433),
         AlphaLevel::TwoPointFivePercent => (-3.1175, -4.53235, -9.8824, -57.7669),
         AlphaLevel::FivePercent => (-2.86154, -2.86154, -4.234, -40.04),
         AlphaLevel::TenPercent => (-2.56677, -1.5384, -2.809, 0.),
     };
+    return calculate_t_stat_from_estimators(t, u, v, w, sz);
+}
+
+pub fn no_constant_no_trend_critical_value<F: Float>(
```

Comment:
Please add the docstring.

Reply:
my bad I didn't format one of the functions properly I just pushed again

Reply:
thanks